### ISSUE TITLE
buddy list empty after session starts

### ISSUE THREAD

**Issue by steveatinfincia** [unixpickle/LibOrange#2]
seems to happen almost at certain times of day for a few hours.

getting this in the console:

Unfatal ERROR: Got invalid NickWInfo from OSERVICE

Nick info dump: <>

**Comment by unixpickle** [comment]
How often are you using LibOrange to sign in to one AIM account? I am asking this question because after signing in a few times during a short period of time, AOL stops allowing you to log in through that account.  Later today or tomorrow I will add some more debug code so that I can get a better idea of the problem that you are having, because right now it seems to be a problem with AOL's backend servers.

When you get this error with LibOrange, try using the same AIM account through iChat and see if it works.  If it doesn't, we know that it's an issue with AOL's backend or your specific AIM account rather than with LibOrange.  I will contact you when I have added more debug code so that we can figure out how to fix this issue.

**Comment by unixpickle** [comment]
I actually just stumbled upon the problem by accident.  It turns out that the issue you're facing was due to the fact that you were compiling for the "Release" target instead of debug.  Why might this be an issue?  Because in SNAC.m I have some code written in an NSAssert, and NSAssert statements do not get compiled for "Release." I have fixed the problem and pushed my changes.  I suggest that everyone pulls these changes (even if you are never going to Release your product to the public).
